File: coffee-system/src/main/java/com/jesperancinha/coffee/system/launcher/CoffeeParadigmsMain.java
package com.jesperancinha.coffee.system.launcher;

import jakarta.xml.bind.JAXBException;
import org.kohsuke.args4j.CmdLineException;
import org.kohsuke.args4j.CmdLineParser;
import org.xml.sax.SAXException;

import java.io.FileNotFoundException;

/**
 * Created by joaofilipesabinoesperancinha on 30-04-16.
 */
public class CoffeeParadigmsMain {
    public static void main(String[] args) throws
            CmdLineException,
            FileNotFoundException,
            JAXBException {
        final CoffeParadigmsOptions options = new CoffeParadigmsOptions();
        final CmdLineParser parser = new CmdLineParser(options);
        parser.parseArgument(args);
        options.run();
    }
}

File: coffee-system/src/main/kotlin/org/jesperancinha/coffee/system/CoffeeParadigmsMain.kt
package org.jesperancinha.coffee.system

import jakarta.xml.bind.JAXBException
import picocli.CommandLine
import java.io.FileNotFoundException
import kotlin.system.exitProcess

/**
 * Created by joaofilipesabinoesperancinha on 30-04-16.
 */
@Throws(FileNotFoundException::class, JAXBException::class)
fun main(args: Array<String>) {
    val options = CoffeeParadigmsOptions()
    val executionResultCode = CommandLine(options).execute(*args)
    exitProcess(executionResultCode)
}